src/ddspdrum/module.py

```diff
@@ -163,7 +169,8 @@ def __str__(self):
 
 
 class VCO(SynthModule):
-    """Voltage controlled oscillator.
+    """
```

Comment:
pep8 has a one-liner up top:
`
"""This is what the function does.

   Details go here. Args etc.
`

Reply:
If you want. My preference is we use
```
"""one-liner"""
```

and
```
"""
line1
line2
multiline comment
"""
```

style

Reply:
@maxsolomonhenry both are valid PEP8 according to `black` and `flake8`

Reply:
Works for me.